inc/css-builder.php

```diff
@@ -217,9 +217,11 @@ public function get_css() {
 				continue;
 			}
 
-			if ( $max_res < 1920 ) {
+			if ( $max_res == 0 && ! empty( $min_res ) ) {
```

Comment:
It seems like this is using `0` to say there isn't a `max-width`, which could be confusing, so rather use an empty string. E.g. instead of `0:768` you'll use `:768`.
Then this condition would be `if ( $max_res === '' && $min_res > 0 ) {` or use `$min_res >= 0` if you want to include `0` here